.editorconfig

```diff
@@ -0,0 +1,20 @@
+# EditorConfig
+# http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.js]
+indent_style = tab
+tab_width = 4
+
+[{package.json,.travis.yml}]
+indent_style = space
+indent_size = 2
```

Comment:
What's the basis for these settings? They'd have to be kept in sync with the ESLint config right? (https://github.com/Leaflet/Leaflet/issues/8176 / https://github.com/Leaflet/Leaflet/issues/3870)